bigtable/client/internal/prefix_range_end_test.cc

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/client/internal/prefix_range_end.h"
+
+#include <gmock/gmock.h>
+
+TEST(PrefixRangeEndTest, Simple) {
+  // We assume ASCII in this test, sue me.
```

Comment:
I'm not a lawyer, but I play one on GitHub, so I would recommend rephrasing this as:

```cpp
// This test assumes ASCII.
```

Reply:
Fixed.